pytest_django/plugin.py

```diff
@@ -89,6 +89,14 @@
              'assertNotContains', 'assertNumQueries', 'assertQuerysetEqual',
              'assertRaisesMessage', 'assertRedirects', 'assertTemplateNotUsed',
              'assertTemplateUsed', 'assertXMLEqual', 'assertXMLNotEqual'),
+    # from https://github.com/django/django/blob/stable/1.10.x/django/test/testcases.py#L155
+    #  and https://github.com/django/django/blob/stable/1.10.x/django/test/testcases.py#L797
+    (1, 10): ('assertContains', 'assertFieldOutput', 'assertFormError',
+              'assertFormsetError', 'assertHTMLEqual', 'assertHTMLNotEqual',
+              'assertInHTML', 'assertJSONEqual', 'assertJSONNotEqual',
+              'assertNotContains', 'assertNumQueries', 'assertQuerysetEqual',
+              'assertRaisesMessage', 'assertRedirects', 'assertTemplateNotUsed',
+              'assertTemplateUsed', 'assertXMLEqual', 'assertXMLNotEqual'),
 }
```

Comment:
See my comment, they can be formed dynamically rather than using any duplication in the file or from django itself.